Sources/Siwf/SiwfButton.swift

```diff
@@ -44,7 +44,7 @@ public struct SiwfButton: View {
             }
             .padding(.vertical, 6)
             .padding(.leading, -6)
-            .frame(width: 254)
+            .frame(width: 276)
```

Comment:
with the longer text, we needed a larger button

Reply:
Hmm... This worries me. If we expand (or shrink?) the button text again by changing it at the siwf/assets.json level could this cause issues in older apps with older SDKs. Thoughts?